evidence/pool_test.go

```diff
@@ -96,7 +96,7 @@ func TestProposingAndCommittingEvidence(t *testing.T) {
 	assert.False(t, pool.IsCommitted(evidence))
 
 	// test evidence is proposed
-	proposedEvidence := pool.PendingEvidence(-1)
+	proposedEvidence := pool.PendingEvidence(1)
```

Comment:
```suggestion
	proposedEvidence := pool.AllPendingEvidence()
```

?

Reply:
Sure, I can change it to that - but there is only one piece of evidence added (or at least should be)

Reply:
Okay then